### PR TITLE
Add DUSD Staking on Davos Protocol 

### PR DESCRIPTION
**Pull request by 0nchainanalysys**
- **Davos Protocol**:

- **[Project Website](https://davos.xyz/)**:

- **Description**: (The staking yield of DUSD token is simple. When a user collateralizes their MATIC and borrows DUSD, a percentage of the MATIC gets converted into ankrMATIC reward-bearing tokens. Then, the reserve of ankrMATIC accumulates in value over time and the rewards get converted back into DUSD and a proportion is granted to stakers and liquidity providers.

Another aspect in which the protocol generates yield for DUSD stakers or liquidity prooviders stems from the borrowing interest paid to the protocol. The borrowing rate for DUSD is 2% and these fees get collected in the Davos Protocol Revenue pool and is then distributed respectively.

Yield is also generated through liquidity pools. If you want to maximize your yield from DUSD you can also add DUSD liquidity pairs to the available liquidity pools ([Uniswap](https://info.uniswap.org/#/polygon/pools/0x3ef7f8fa80e622f358283145172143116c61524d) and [Quickswap](https://quickswap.exchange/#/analytics/v3/pair/0xfb0bc232cd11dbe804b489860c470b7f9cc80d9f)) and can receive rewards from the yield farming process on these decentralized exchanges.)

- **Please check if the PR fulfills these requirements**

* [x] The projects name, twitter, URL etc have been filled out in `index.ts`
* [x] Tests for the new features have been added (for bug fixes / features)
* [x] Any required Logos are included in this PR

- **Logo**: ([High res logo compatible with dark and light backgrounds](https://ibb.co/Lk9Sbxc))

- **Other information**: (Any farms or features left to be implemented, addresses to be used for testing, etc)

### CODE REVIEW

File: apps/workshop/projects/DavosProtocol/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type { ProjectDefinitionInterface } from '@defiyield/sandbox';
+
+import { DavosProtocol } from './modules/DUSDStake';
```
Comment (artembondadefiyeld) [inline apps/workshop/projects/DavosProtocol/index.ts:3]: looks like a missing reference

Comment (0nchainanalysys) [reply apps/workshop/projects/DavosProtocol/index.ts:3]: on the second line?

Comment (artembondadefiyeld) [reply apps/workshop/projects/DavosProtocol/index.ts:3]: but the example module has `abi` inside, whereases the module in the PR doesn't.  Please make sure the module works fine running:
 yarn workshop start [integration_name]

File: apps/workshop/projects/DavosProtocol/modules/DUSDStake.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  async fetchPools({ tokens, ethcallProvider, ethcall, BigNumber, axios }) {
+    const [token] = tokens;
+    const contract = new ethcall.Contract(stDUSDAddress, stDUSDAbi);
+
+    const [totalSupply] = await ethcallProvider.all<typeof BigNumber>([contract.totalStaking()]);
+    const tvl = (Number(totalSupply.toString()) / 10 ** token?.decimals) * (token?.price || 0);
+
+    let apr = 0;
+    try {
+      const response = await axios.get('https://stake.ly/api/stats/stDUSD/apr');
```
Comment (artembondadefiyeld) [inline apps/workshop/projects/DavosProtocol/modules/DUSDStake.ts:25]: Does the URL work fine? 
image

File: apps/workshop/projects/DavosProtocol/index.ts (hunk 1 above)
Comment (0nchainanalysys) [reply apps/workshop/projects/DavosProtocol/index.ts:3]: apart from the abis module, is there anything else required?

**Pull request closed without merging**